### DISCUSSION TITLE
FEATURE REQUEST: FEWER RENTAL REFUNDS - Direct unrented miners to assist rented miners

### DISCUSSION THREAD

**Discussion by shieldsrholding** [RainbowMiner/RainbowMiner#2714] [Ideas]
I get a lot of end of rental refunds due to very high difficulty at some pools.  It would be nice if un-rented miners could visit rented miners and beef up the hashrate in order to reduce end of rental refunds.  It seems all the pieces of the puzzle are there.  Check the hashrate of the miner.  Check the promised rate on MRR.  If it is less than promised, an un-rented miner joins and brings the rental up to par so that rental closes at or above 100%..

It would be really nice if this was just a simple switch ... HASHRATE ASSIST: y/n

**Comment by RainbowMiner** [comment]
Nice idea. I will think about it.
